src/scss/_learnmore.scss

```diff
@@ -0,0 +1,8 @@
+a {
+  text-decoration: none;
+  color: #0192b5;
+}
+
+a:hover {
+  text-decoration: underline;
+}
```

Comment:
These are a little too general, which you'll encounter first-hand as we try to merge this feature in with everyone else's work. You also seem to be erring on the side of caution here by creating a ton of teeny little files for each piece of the page. Sufficient to have a single SCSS file for _just your feature._